tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -39,7 +40,7 @@
 from tensorflow.python.training import slot_creator
 
 
-def DynamicEmbeddingOptimizer(self, bp_v2=None):
+def DynamicEmbeddingOptimizer(self, bp_v2=None, dist_sync=None):
```

Comment:
‘dist_sync’ is a little confusing words. Maybe 'synchronous' or 'use_allreduce' would be better?
BTW, the default should be False?

Reply:
change to `horovod_synchronous` here.

Reply:
The name of `horovod_synchronous` may bind the API to horovod only. `synchronous` might be enough?